frontend/workflows/envoy/src/remote-triage/dashboard.tsx

```diff
@@ -47,20 +47,22 @@ interface FeaturedSummaryProps {
 const FeaturedSummary = ({ summary }: { summary: FeaturedSummaryProps }) => {
   const values = summary?.data?.map(d => d.value);
   const total = values.reduce((t, value) => t + value);
```

Comment:
I know this isn't part of the PR, but I'd adjust this a bit. We're iterating over values to pull them out to then iterate over them for a total.

```suggestion
  const total = (summary?.data || []).reduce((t, { value = 0 }) => t + value, 0);
```